backend/experiment/management/commands/createruleset.py

```diff
@@ -27,10 +27,10 @@ def create_block_rule_class(self, ruleset_name):
         ruleset_name_snake_case, ruleset_name_snake_case_upper, ruleset_name_pascal_case = self.get_ruleset_name_cases(ruleset_name)
 
         # Create a new file for the block rules class
-        filename = f"./experiment/rules/{ruleset_name_snake_case}.py"
+        filename = join('.', 'experiment', 'rules', ruleset_name_snake_case + '.py')
```

Comment:
```suggestion
        filename = join(".", "experiment", "rules", f"{ruleset_name_snake_case}.py")
```

I think the f syntax is more readable than `+` for string formatting.